src/js/select2/core.js

```diff
@@ -563,11 +564,11 @@ define([
     this._syncS = null;
 
     this.$element.off('.select2');
-    this.$element.attr('tabindex',
-    Utils.GetData(this.$element[0], 'old-tabindex'));
+    this.$element[0].setAttribute('tabindex',
+      Utils.GetData(this.$element[0], 'old-tabindex'));
 
     this.$element[0].classList.remove('select2-hidden-accessible');
-    this.$element.attr('aria-hidden', 'false');
+    this.$element[0].setAttribute('aria-hidden', 'false');
```

Comment:
Is the adjustment in whitespace before `Utils.GetData` here intentional?  It is logically contained within a staement that has been modified, so I can understand if the idea is to communicate/include that.